### PR TITLE
Fix table sorting of generators

### PR DESCRIPTION
**Pull request by htanjo**
The table sorting seems broken again.
http://yeoman.io/generators/

This patch recovers sort feature, but the order of "last update" is not correct because [List.js](http://www.listjs.com/) sorts contents alphabetically. (e.g. "2 days ago" > "2 months ago" > "3 days ago")

Lits.js has a tricky [workaround](https://github.com/javve/list.js/issues/203#issuecomment-33517804) and  [feature request](https://github.com/javve/list.js/issues/35) to improve the data handling.

What do you think?
If the workaround is ok, I will update my PR to fix order.

### CODE REVIEW

File: app/assets/js/generator.js [hunk 1]
```diff
@@ -63,7 +63,7 @@
 
         var list = new List('plugins-all', {
           valueNames: [
-            'name',
+            'name-website',
```
Comment (eddiemonge) [inline app/assets/js/generator.js]: why this change?

Comment (htanjo) [inline app/assets/js/generator.js]: List.js checks the inner content for sorting.
`<span class="name-website">` is more specific than `<td class="name">` and this improves the performance.

**Comment by eddiemonge** [comment]
thanks

**Comment by htanjo** [comment]
No problem!
